osier/models/capacity_expansion.py

```diff
@@ -71,39 +63,39 @@ class CapacityExpansion(ElementwiseProblem):
     allow_blackout : boolean
         If True, a "reliability" technology is added to the dispatch model that will
         fulfill the mismatch in supply and demand. This reliability technology
-        has a variable cost of 1e4 $/MWh. The value must be higher than the 
-        variable cost of any other technology to prevent a pathological 
+        has a variable cost of 1e4 $/MWh. The value must be higher than the
+        variable cost of any other technology to prevent a pathological
         preference for blackouts. Default is True.
```

Comment:
If `allow_blackout` is true, then it will fulfill the mismatch in supply and demand? I think it would be more intuitive if this happened if `allow_blackout` was false.

Reply:
Since this dispatch model is an LP problem, demand = supply at every time step. Otherwise, the model is infeasible. If I want to model situations where demand cannot be met (i.e. "allowing blackouts" to occur), I add a dummy technology  whose fuel cost corresponds to the "value of lost load." I can add a separate section in the docs to explain this if that would make you happy! (If this is what you want to see, please make an issue!)

Reply:
I think I understand a bit better. So if `allow_blackout` is `True`, you add a "reliability" technology to capture the value of the lost load. If `allow_blackout` is `False`, the "reliability" technology isn't included. In that case, is demand limited or is supply raised to prevent a blackout from occurring, or does supply not meet demand in that case?

Reply:
In that case the model is infeasible because supply cannot meet demand.